oak_utils/src/lib.rs

```diff
@@ -163,6 +163,20 @@ impl prost_build::ServiceGenerator for OakServiceGenerator {
     }
 }
 
+/// Options for generating Protocol buffer Rust types.
+pub struct ProtoOptions {
+    /// Generates service code for inter-node communication. Default: **true**.
+    pub generate_services: bool,
+}
+
+impl Default for ProtoOptions {
+    fn default() -> ProtoOptions {
+        ProtoOptions {
+            generate_services: true,
```

Comment:
I find this a bit confusing, also that it needs to be kept in sync with the comment above (what does it mean for a bool field to have a default of true? really it's just this method that sets it to true, so perhaps the comment should be here). I think having an `Option` would be more explicit, or just require callers to set it to true (I don't think we need to worry about that kind of repetition).

Reply:
I agree it would be nice to put the explanation of the default value next to where that is actually defined, but as a user you would probably only read the documentation on each of the fields, having to drill down into the `Default` impl is not very intuitive here.

The `Option` strategy is interesting, but that would mean we do things differently between `ProtoOptions` and `CodegenOptions`, which we would probably want to avoid if possible. What do you think?

Reply:
I still think the default should be false, and we set it to true where needed :) But I'll leave it to you to decide.